bundles/CoreBundle/DependencyInjection/Configuration.php

```diff
@@ -1249,6 +1251,70 @@ private function addTargetingNode(ArrayNodeDefinition $rootNode)
             ->end();
     }
 
+    private function addConfigurationSectionsNode(ArrayNodeDefinition $rootNode)
```

Comment:
We don't need that anymore, we're just using the standard Symfony Config approach.